docs/tools-infrastructure/hosted-gateway.md

```diff
@@ -4,3 +4,46 @@ sidebar_position: 3
 
 # Hosted Gateway
 
+The Obscuro Gateway is a critical application that facilitates communication between an Obscuro node and various tools that require a connection to it, such as MetaMask. Due to the encryption of data within an Obscuro node, direct communication is not feasible.
+
+The program conforms to the Ethereum JSON-RPC specification ([Ethereum JSON-RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/eth1.0-apis/assembled-spec/openrpc.json)) and also supports additional APIs to ensure compatibility with popular tools like MetaMask.
+
+You have the flexibility to host the Obscuro Gateway yourself or use one of the hosted gateways if you choose to join Obscuro. If security is your primary concern, you can run and use the program independently. The diagram below illustrates different usage scenarios, with Bob and Charlie using the hosted version and Alice managing it independently.
```

Comment:
I would replace 'if security is your primary concern" with "you have the option to."